test/unit/api-config-test.js

```diff
@@ -39,6 +39,13 @@ define([
                 return x;
             }
 
+            it('should have default width of 640 and height of 360', function() {
+                let x = testConfig();
```

Comment:
I was going to do a `const` but wanted to be consistent with the rest of the file. 

Will update